featuretools/tests/conftest.py

```diff
@@ -746,6 +747,52 @@ def rolling_outlier_series_pd():
     )
 
 
+@pytest.fixture
+def postal_code_dataframe():
+    df = pd.DataFrame(
+        {
+            "string_dtype": pd.Series(["90210", "60018", "10010", "92304-4201"]),
+            "int_dtype": pd.Series([10000, 20000, 30000]).astype("category"),
+            "has_nulls": pd.Series([np.nan, 20000, 30000]).astype("category"),
+        },
+    )
+    df.ww.init(
+        logical_types={
+            "string_dtype": "PostalCode",
+            "int_dtype": "PostalCode",
+            "has_nulls": "PostalCode",
+        },
+    )
+    return df
+
+
+@pytest.fixture
+def postal_code_dataframe_pyspark(postal_code_dataframe):
+    ps = pytest.importorskip("pyspark.pandas", reason="Spark not installed, skipping")
+    ans = ps.from_pandas(postal_code_dataframe)
+    return ans
+
+
+@pytest.fixture
+def postal_code_dataframe_dask(postal_code_dataframe):
+    ans = dd.from_pandas(
+        postal_code_dataframe,
+        npartitions=1,
+    ).compute()
```

Comment:
This compute call converts this into a pandas dataframe

Reply:
Another note - in both this fixture and the spark fixture, you are losing the WW initialization. You will either need to reinitialize if that is important or handle initialization for all the fixtures inside the tests.